lib/index.js

```diff
@@ -169,11 +169,17 @@ class Configurator {
 			}
 		};
 
-		if (this.opts.include) {
-			this.opts.include.forEach(
-				path => babelConfig.include.push(new RegExp(path))
-			);
-		}
+		// TODO: these modules need to be transpiled
+		// NOTE: these are npm modules, babel has a freak out if you just include all of /node_modules/
+		const es6Modules = [
+			'@financial-times/n-card',
+			'@financial-times/n-image',
+			'@financial-times/n-section',
+			'@financial-times/n-ui'
```

Comment:
Al this says is 'if these modules are required, babelify them'. If an app doesn't use these modules, then nothing happens. 'Default' probably isn't the right word, these are just modules that aren't transpiled